gui/mozregui/wizard.py

```diff
@@ -147,6 +148,18 @@ def _set_fetch_config(self, index):
             self.ui.url.show()
             self.ui.url_label.show()
 
+        # lang only makes sense for firefox-l10n, and repo doesn't
+        if app_name == "firefox-l10n":
+            self.ui.lang.show()
+            self.ui.lang_label.show()
+            self.ui.repository.hide()
+            self.ui.repository_label.hide()
+        else:
+            self.ui.lang.hide()
```

Comment:
I wonder if we would be better off making it insensitive/grey rather than hiding it? I think it would be a little less confusing than have UI elements appear/disappear (I believe this is the recommended practice according to many UI style guides)

Reply:
Ah, yes I like this better.  I wasn't happy with the disappearing myself (especially since the hidden ones seem to leave their margins around, or something else that affects the vertical gaps between options).

Should do a follow up to do the same with the rest of the options (arch, URL etc.)